src/main/java/com/google/firebase/projectmanagement/Scheduler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright  2019 Google Inc.
```

Comment:
nit: There is an extra space between Copyright and 2019.

Reply:
Done